### ISSUE TITLE
Khronos GLSL grammar 4.60.7 has typo in "layout qualifier" section

### ISSUE THREAD

**Issue by AndrewRayCode** [KhronosGroup/GLSL#161]
In [The OpenGL® Shading Language, Version 4.60.7](https://www.khronos.org/registry/OpenGL/specs/gl/GLSLangSpec.4.60.pdf), on page 67, there is this line:

<img width="456" alt="Screen Shot 2021-06-15 at 12 09 41 AM" src="https://user-images.githubusercontent.com/79215/122008526-05108f00-cd6e-11eb-91f3-f93fae652616.png">

I believe there is a typo here. Can you spot it?

(Scroll down for the spoiler)

I believe type typo is on the last line. `int vec4 v;` should be `in vec4 v;` I don't believe the former is valid GLSL.

**Comment by gnl21** [comment]
> I believe type typo is on the last line.

Yes, this looks clearly incorrect. I'll fix it in the next version of the specification. Thanks.

> I'm not sure if this is the right place to file issues with the core grammar spec. Is there a better place?

This is the right place for filing spec issues.

**Comment by dj2** [comment]
This appears to be fixed in the latest spec, can this be closed?
